OpenEdXMobile/src/main/java/org/edx/mobile/view/dialog/RatingDialogFragment.java

```diff
@@ -0,0 +1,119 @@
+package org.edx.mobile.view.dialog;
+
+import android.app.Dialog;
+import android.content.Context;
+import android.content.DialogInterface;
+import android.os.Bundle;
+import android.support.v4.app.FragmentActivity;
+import android.support.v7.app.AlertDialog;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.widget.Button;
+import android.widget.RatingBar;
+
+import com.google.inject.Inject;
+
+import org.edx.mobile.BuildConfig;
+import org.edx.mobile.R;
+import org.edx.mobile.base.MainApplication;
+import org.edx.mobile.module.prefs.PrefManager;
+import org.edx.mobile.view.Router;
+
+import roboguice.fragment.RoboDialogFragment;
+
+public class RatingDialogFragment extends RoboDialogFragment implements AlertDialog.OnShowListener,
+        RatingBar.OnRatingBarChangeListener {
+    @Inject
+    private Router mRouter;
+    private AlertDialog mAlertDialog;
+    private RatingBar mRatingBar;
+
+    public static RatingDialogFragment newInstance() {
+        return new RatingDialogFragment();
+    }
+
+    @Override
+    public Dialog onCreateDialog(Bundle savedInstanceState) {
+        final LayoutInflater inflater = (LayoutInflater) getContext().getSystemService(Context.LAYOUT_INFLATER_SERVICE);
+        final View view = inflater.inflate(R.layout.fragment_dialog_rating, null);
+        mRatingBar = (RatingBar) view.findViewById(R.id.ratingBar);
```

Comment:
nit: We try to use data binding. There only be a minimal improvement here so it's a nit.